app/services/permission_service.rb

```diff
@@ -37,7 +38,7 @@ def allow?(controller, action)
       return true if controller == 'cart_cars'   && action.in?(%w(create destroy update))
       return true if controller == 'cart'        && action == 'show'
       return true if controller == 'orders'      && action.in?(%w(index create show))
-      return true if controller == 'users'       && action.in?(%w(show edit update))
+      return true if controller == 'users'       && action.in?(%w(show edit update profile))
```

Comment:
Line is too long. [92/80]<br>Operator `&&` should be surrounded with a single space.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.